Apps/WebClient/src/ClientApp/test/reports.test.ts

```diff
@@ -0,0 +1,344 @@
+import "@/plugins/inversify.config";
+import { createLocalVue, shallowMount, Stubs, Wrapper } from "@vue/test-utils";
+import VueContentPlaceholders from "vue-content-placeholders";
+import VueRouter from "vue-router";
+import Vuex from "vuex";
+
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import { ILogger } from "@/services/interfaces";
+import { GatewayStoreOptions } from "@/store/types";
+import ReportsView from "@/views/reports.vue";
+
+import { StoreOptionsStub } from "./stubs/store/store";
+import container from "@/plugins/inversify.container";
```

Comment:
This file isn't linted.  The imports aren't sorted and there are uses of `var` instead of `const`/`let`.

Reply:
all clean now!